src/ray/core_worker/task_interface.cc

```diff
@@ -185,7 +186,7 @@ Status CoreWorkerTaskInterface::SubmitActorTask(ActorHandle &actor_handle,
   std::unique_lock<std::mutex> guard(actor_handle.mutex_);
   // Build actor task spec.
   const auto actor_creation_dummy_object_id =
-      ObjectID::FromBinary(actor_handle.ActorID().Binary());
+      ObjectID::GenerateActorDummyObjectId(actor_handle.ActorID());
```

Comment:
Same thing here, this doesn't seem to match the current behavior in the Python worker. Add a `TODO` to fix this...

Reply:
Sorry, I can't see any issue or (difference from python side)  about this change. Could you tell more details?